app/components/Content/Widget.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { Card } from '@aragon/ui'
+import { Card, GU } from '@aragon/ui'
 import MarkdownPreview from '../Widget/Markdown/Preview'
-import Votes from './Votes'
+import Voting from '../Widget/Voting/Voting'
```

Comment:
`/* super-opinionated-comment-start */`

The widget is named "Latest Votes" so the component should be named `LatestVotes` or just `Votes`, why complicate things when we can use that simple approach?

Now we need to use `Voting` to refer to the `Latest Votes` widget, that furthermore, collides with the `Voting` app name... I suggest taking the easier path for everybody, name components are they are labelled in the designs, same for buttons, or any other component that has a label.

`/* super-opinionated-comment-end */`